security-privacy-questionnaire.md

```diff
@@ -30,50 +30,55 @@ Additionally, user agents could also choose to persist the last directory a file
 
-### 2.11. Does this specification allow an origin some measure of control over a user agent’s native UI?
+### 2.11. Do features in this specification allow an origin some measure of control over a user agent’s native UI?
 
 The origin can pop up native file or directory pickers, and have some control over what appears inside that native UI (e.g. accepted file types, starting directory and suggested file names), but that control is very limited. The spec does put limitations on what is allowed as an accepted file type and suggested file name to limit the security impact of allowing websites to control the native UI. User agents are expected to employ similar mechanisms to sanitize the sugessted file names as are used to sanitize for example suggested file names in `<a download="foo.ln">` today.
 
-### 2.12. What temporary identifiers might this this specification create or expose to the web?
+### 2.12. What temporary identifiers do the features in this specification create or expose to the web?
 
 The `getUniqueId()` method will create a temporary unique identifier for a given handle. This ID will become invalid if the user clears storage for the site.
 
 ### 2.13. How does this specification distinguish between behavior in first-party and third-party contexts?
 
 It is expected that user agents do not allow third-party contexts to prompt for any kind of access using this API. I.e. third-party contexts can potentially access files or directories that their origin was already granted access to in a first-party context (by sharing handles via IndexedDB or postMessage), but can't trigger any new file/directory pickers or permission requests.
 
-### 2.14. How does this specification work in the context of a user agent’s Private Browsing or "incognito" mode?
+### 2.14. How do the features in this specification work in the context of a browser’s Private Browsing or Incognito mode?
 
 The feature will work mostly the same as in regular mode, except no handles or permission grants will be persistent. Web sites can use this API to store data to disk even in private browsing mode, but to later be able to read this data again (either from private browsing or regular mode), the user would have to explicitly re-pick the same file or directory.
 
-### 2.15. Does this specification have a "Security Considerations" and "Privacy Considerations" section?
+### 2.15. Does this specification have both "Security Considerations" and "Privacy Considerations" sections?
 
 Yes.
 
-### 2.16. Does this specification allow downgrading default security characteristics?
+### 2.16. Do features in your specification enable origins to downgrade default security protections?
 
 No.
 
+### 2.17. How does your feature handle non-"fully active" documents?
+
+This feature uses a locking system to prevent data races when modifying/reading a handle. Locks are acquired on the handle. These locks are held by the user agent, not the platform.
```

Comment:
> Locks are acquired on the handle.

Well, [not exactly](https://github.com/whatwg/fs/pull/131)... This doesn't add much IMHO so I would recommend leaving it out

> These locks are held by the user agent, not the platform.

[Not necessarily](https://crbug.com/1240780). This is up to the user agent's implementation and is subject to change. Should probably also leave this out